build.gradle

```diff
@@ -16,6 +16,10 @@ buildscript {
     }
 }
 
+plugins {
+  id("org.sonarqube") version "4.3.1.3277"
+}
+
```

Comment:
As I understand, the problem was here that before the plugins part were other configurations, but "buildscript" allowed